src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,9 +2,12 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int TAB_SIZE = 12;
+	private static final int TOTAL = 11;
+	private static final int CHECK = -1;
```

Comment:
To nie jest ta sama stała o wartości -1, powinna być osobna w metodach peekaboo i countOut.